ios/Video/RCTVideo.swift

```diff
@@ -189,16 +189,23 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
     @objc func togglePlaybackController(){
         _rctPlaybackControls?.toggleControlVisibility(visible: true)
     }
-
-    func resetWrapperViewController() -> UIViewController{
-        // Clean wrapper
-        _wrapperViewController.removeFromParent()
-
+    
+    func removeWrapperController(){
+        // Toggle off full screen presentation
+        self.setFullscreen(false)
```

Comment:
Nit: no need to add `self.` for function calls (unless they are in a closure where you have to worry about retain cycles)